### DISCUSSION TITLE
关于给fcitx5-configtool提PR如何更新翻译

### DISCUSSION THREAD

**Discussion by asterwyx** [fcitx/fcitx5#994] [Q&A] [answered]
近期我给fcitx5-configtool提了一个PR https://github.com/fcitx/fcitx5-configtool/pull/76 ，其中新增了一条文案，请问应该如何更新翻译？Messages.sh脚本似乎并不能在我机器上正常运行，使用yay也没有搜索到"gen_pot"这个可执行文件在哪儿。transifex上我似乎也不能加入project，使用xgettext等工具进行本地更新的效果也没有达到预期（或许是我的使用方式有问题）。请问我应该通过哪种方式新增文案呢？

**Comment by wengxt** [comment, marked as answer]
翻译都在 https://explore.transifex.com/fcitx/fcitx/ 上进行，合并之后等待同步，不要自己更新翻译
